test/fixtures/vote_account_histories.yml

```diff
@@ -7,6 +7,9 @@ one:
   last_vote: 1
   activated_stake: 1_000_000_000
   credits: 1
+  credits_current: 193916
+  slot_index_current: 299198
+  skipped_vote_percent_moving_average: 0.1
```

Comment:
This is ultimately up to @brianlong, but I highly recommend using `FactoryBot` instead of the Rails fixtures. Rails fixtures don't use or call ActiveRecord validations or callbacks, so the objects you create via fixtures can end up looking completely different than what would be permissable or happen in the real app.